### DISCUSSION TITLE
Hooking into the lighting engine

### DISCUSSION THREAD

**Discussion by tristan-f-r** [Minestom/Minestom#615] [Help] [answered]
Hello! I was wondering, how can I override the default lighting system in Minestom with my own?

**Comment by mworzala** [comment, marked as answer]
There is no "default lighting system" currently. Light is loaded from Anvil worlds, and you can set the light arrays manually in the `Section`s.

I am (sort of, I have a lack of time the last few weeks) working on an internal lighting system to mimic vanilla. The plan is to keep it completely internal for the moment however.
